Description: Add extra lang files shipped in data to Makefile
Author: David Hewitt <davidmhewitt@gmail.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/language-specs/Makefile.am
+++ b/data/language-specs/Makefile.am
@@ -5,6 +5,8 @@
 	actionscript.lang	\
 	ada.lang		\
 	ansforth94.lang		\
+	apache.lang		\
+	asm.lang		\
 	asp.lang		\
 	automake.lang		\
 	awk.lang		\
@@ -12,29 +14,34 @@
 	bibtex.lang		\
 	bluespec.lang		\
 	boo.lang		\
+	cfml.lang		\
 	cg.lang			\
 	changelog.lang		\
 	chdr.lang		\
 	c.lang			\
 	cmake.lang		\
 	cobol.lang		\
+	coffee.lang		\
 	cpphdr.lang		\
 	cpp.lang		\
 	csharp.lang		\
 	css.lang		\
 	csv.lang		\
+	cucumber.lang	\
 	cuda.lang		\
 	def.lang		\
 	desktop.lang		\
 	diff.lang		\
 	d.lang			\
 	docbook.lang		\
+	dokuwiki.lang	\
 	dosbatch.lang		\
 	dot.lang		\
 	dpatch.lang		\
 	dtd.lang		\
 	dtl.lang		\
 	eiffel.lang		\
+	elixir.lang		\
 	erlang.lang		\
 	fcl.lang		\
 	forth.lang		\
@@ -46,9 +53,11 @@
 	glsl.lang		\
 	go.lang			\
 	groovy.lang		\
+	gsp.lang		\
 	gtk-doc.lang		\
 	gtkrc.lang		\
 	haddock.lang		\
+	haml.lang		\
 	haskell.lang		\
 	haskell-literate.lang	\
 	haxe.lang		\
@@ -72,6 +81,7 @@
 	lua.lang		\
 	m4.lang			\
 	makefile.lang		\
+	mako.lang		\
 	mallard.lang		\
 	markdown.lang		\
 	matlab.lang		\
@@ -102,15 +112,20 @@
 	puppet.lang		\
 	python3.lang		\
 	python.lang		\
+	rhtml.lang		\
 	R.lang			\
 	rpmspec.lang		\
 	rst.lang		\
 	ruby.lang		\
+	ruby_on_rails.lang	\
 	rust.lang		\
+	sass.lang		\
 	scala.lang		\
 	scheme.lang		\
 	scilab.lang		\
+	scss.lang		\
 	sh.lang			\
+	slim.lang		\
 	sml.lang		\
 	sparql.lang		\
 	sql.lang		\
